src/main/java/io/cdap/plugin/AddField.java

```diff
@@ -138,7 +138,7 @@ public Schema getOutputSchema(GetSchemaRequest request) {
   private Schema getOutputSchema(Schema inputSchema, Conf config) {
     List<Schema.Field> fields = new ArrayList<>(inputSchema.getFields().size() + 1);
     fields.addAll(inputSchema.getFields());
-    fields.add(Schema.Field.of(config.fieldName, Schema.nullableOf(Schema.of(Schema.Type.STRING))));
+    fields.add(Schema.Field.of(config.fieldName, Schema.of(Schema.Type.STRING)));
```

Comment:
why is this change needed?

Reply:
@CuriousVini As per requirements of this task https://issues.cask.co/browse/PLUGIN-235